hydro/distribution.yaml

```diff
@@ -3883,6 +3883,17 @@ repositories:
       type: git
       url: https://github.com/UC3MSocialRobots/maggie_devices_msgs.git
       version: hydro-devel
+    release:
+      packages:
+      - eyelids_msgs
+      - ir_controller_msgs
+      - maggie_devices_msgs
+      - motor_controller_msgs
+      - rfid_msgs
```

Comment:
Since all packages are in a flat namespace we try to be careful about generic names. rfid_msgs seems like something that once reviewed by the community might be promoted to be inside common_msgs. However if you use the `rfid_msgs` name in your release either it would have to be your implementation  or we would have to pick a different name. There are already other rfid messages defined such as: http://wiki.ros.org/hrl_rfid 

Also when searching for other existing rfid messages I note you've picked the name [rfid](http://wiki.ros.org/rfid) for another package. 

This will be packaged into ros-indigo-rfid so someone coming along may try to install that to add rfid functionality to their robot. And they'll be surprised to find a driver package for ["the Social Robot Maggie"](https://github.com/UC3MSocialRobots/maggie_devices/tree/hydro-devel/rfid)

In general please review the [Package Naming Conventions](http://wiki.ros.org/ROS/Patterns/Conventions) Anything which is robot or family of robots specific should use a prefix to indicate as such.